### DISCUSSION TITLE
[QUESTION]: unlink not working

### DISCUSSION THREAD

**Discussion by simplecommerce** [GrapesJS/grapesjs#5986] [Q&A]
Hi,

I had a quick question, I was wondering why `unlink` command doesn't always work on a A tag?

I thought it was my code, but I am able to reproduce it in the demo and I don't know why its doing this.

https://grapesjs.com/demo

Start by clearing the canvas.

If you drag the Link block in a container and then double click to select all of the text and click unlink, it doesn't work.

I noticed that if I link a part of the text and then highlight the whole text and unlink, it always works.

I was wondering if anyone had the same issue, and if yes how did you fix it?

I am using grapesjs@0.21.10

Thanks!

**UPDATE**

Ok, I understand what its doing.
Basically, when a Link component is alone, it can't unlink it, since it has to exist inside a Text Component or other Component.
